javascript/README.md

```diff
@@ -643,3 +650,74 @@ const { left, top } = processInput(input);
 ```
 
 [⇧ top](#javascript-guide)
+
+---
+
+## Strings
+
+### 5.1 Quotes
+
+Use single quotes `''` for strings.
```

Comment:
Here's what Prettier does:
```suggestion
Use single quotes `''` for strings. When strings have a literal `'` in them, use `"` for the ends of the strings instead.
```
(I think I worded that pretty badly)

I think we need to keep this section because Prettier handles conversion between `"` and `'`, but it won't switch a template literal to a string

Reply:
@calebeby Ah, got it. I've updated the PR. Let me know if I captured your suggestion properly. 👍